src/EventSubscriber/CartEventSubscriber.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Drupal\commerce_option\EventSubscriber;
+
+use Drupal\commerce_cart\Event\CartEvents;
+use Drupal\commerce_price\Calculator;
+
+use Drupal\Core\TempStore\PrivateTempStoreFactory;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Class CartEventSubscriber.
+ */
+class CartEventSubscriber implements EventSubscriberInterface {
+
+  /**
+   * Drupal\Core\TempStore\PrivateTempStoreFactory definition.
+   *
+   * @var \Drupal\Core\TempStore\PrivateTempStoreFactory
+   */
+  protected $tempstorePrivate;
```

Comment:
Really nitpicking here, but I think the variable name should be `tempStorePrivate`